reference/build_helpers/cmake.rst

```diff
@@ -133,6 +133,23 @@ An example of the value of this property could be:
 
     --config Release
 
+parallel
+++++++++
+
+**Defaulted to**: ``True``
+
+Run CMake process in parallel for compilation, installation and testing. This attribute gets the value for the class constructor and
+appends the ``/MP`` flag in the case of the Visual compiler. However, the parallel executing can be changed for testing for example:
```

Comment:
Rephrase this. The flag `parallel` translate into the proper command line argument, for Unix Makefiles it is `-jX` and for MSBuild it is `/m:X`. I think that the `/MP` is a definition you could have in your CMakeLists, but not an argument in the cli.

Reply:
I was trying to reference this https://github.com/conan-io/conan/blob/a52dcdc87dd8146b661ed650da616b9fcd2371e3/conans/client/build/cmake_flags.py#L259-L258 but you are right too